README.md

```diff
@@ -136,14 +136,14 @@ the `--env-file` flag.
 The most important variables to set are:
 
  * CONFIG_QUEUES_MASTER_NAME: Set this to the [`redis://` URL](http://www.iana.org/assignments/uri-schemes/prov/redis)
-   of where the Redis Storage has been installed
- * CONFIG_REDIS_PROXY: Set this to the [`redis://` URL](http://www.iana.org/assignments/uri-schemes/prov/redis)
    of where the Redis Resque has been installed
+ * CONFIG_REDIS_PROXY: Set this to the [`redis://` URL](http://www.iana.org/assignments/uri-schemes/prov/redis)
```

Comment:
I believe these ones were swapped... `CONFIG_REDIS_PROXY` is for storage, and `CONFIG_QUEUES_MASTER_NAME` is for Resque